### PR TITLE
enhanced configuration via rahmen.toml file, ramped up version number a notch

### PR DESCRIPTION
**Pull request by sys3175**

### CODE REVIEW

File: src/provider.rs [hunk 1]
```diff
@@ -11,6 +11,7 @@ use rexiv2::Metadata;
 
 use crate::config::Element;
 use crate::errors::{RahmenError, RahmenResult};
+use serde::__private::Option::Some;
```
Comment (antiguru) [inline src/provider.rs]: This import seems to be out of place.

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rahmen"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Moritz Hoffmann <antiguru@gmail.com>"]
```
Comment (antiguru) [inline Cargo.toml]: Feel free to add yourself here.

**Review by antiguru — CHANGES_REQUESTED** [review]
Thanks a lot, overall it looks great!

File: src/provider.rs (hunk 1 above)
Comment (sys3175) [reply src/provider.rs]: well, no idea where it came from, and it seems to work regardless, so I'm fine with removing it.

Comment (sys3175) [reply src/provider.rs]: maybe it got added by some Intellij magick.

**Review by antiguru — APPROVED** [review]

**Pull request merged**